apps/game/server/twitter/twitter.service.ts

```diff
@@ -6,12 +6,15 @@ import { reportTweetToDiscord } from '../misc/discord';
 import { PromiseEventResp, PromiseRequest } from '../lib/PromiseNetEvents/promise.types';
 import { getDefaultProfileNames } from '../players/player.utils';
 import { checkAndFilterImage } from './../utils/imageFiltering';
+import { tweetDiscordPost } from '../lib/http-service';
 
 class _TwitterService {
   private readonly twitterDB: _TwitterDB;
+  private readonly DISCORDWH: string;
```

Comment:
Can you change this to `DISCORD_WEBHOOK` the abbreviation might confuse some

Reply:
changed